scripts/kind-e2e/lib_operator_deploy_subm.sh

```diff
@@ -30,13 +30,13 @@ function deploytool_prereqs() {
 function setup_broker() {
     context=$1
     echo Installing broker on $context.
```

Comment:
unrelated to this patch, but before I forget, we need to add "kubectl config use-context $context" here, otherwise if we run the deploy a second time it will fail here, because the config files are at some point modified for an specific cluster config, and then it doesn't work here (using cluster1 here, but context has been set to cluster3 on all config files after a first run)

Reply:
This was done elsewhere, has since been added via rebase.